chat-plugins/jeopardy.js

```diff
@@ -1,772 +1,725 @@
-		if (this.users.get(user).isAnswered) return output.sendReply("You already have answered this question.");
-
-		--this.remainingAnswers;
-		this.currentAnswerer = user;
-		this.currentAnswer = answer;
-		this.users.get(user).isAnswered = true;
-		this.room.add("" + user.name + " has answered '" + answer + "'.");
-
-		if (answer.toLowerCase() === this.questions.getQuestion(this.currentCategory, this.currentQuestion).answer.toLowerCase()) {
-			this.mark(this.host, true);
+		this.state = 'selecting';
+		let lowest = [];
+		let minpoints;
+		for (let userID in this.players) {
+			let points = this.points.get(this.players[userID]);
+			if (!minpoints) {
+				lowest.push(userID);
+				minpoints = points;
+			} else if (points < minpoints) {
+				lowest = [userID];
+				minpoints = points;
+			} else if (points === minpoints) {
+				lowest.push(userID);
+			}
 		}
-	};
-	Jeopardy.prototype.mark = function (user, isCorrect, output) {
-		if (!this.checkPermission(user, output, 'started', 'host')) return;
-		if (this.currentQuestion === 'final') return this.finalMark(user, isCorrect, output);
-		if (!this.currentAnswer) return output.sendReply("There is no answer to mark right now.");
-
-		let userData = this.users.get(this.currentAnswerer);
-		let points = this.isDailyDouble ? userData.wager : this.questions.getQuestion(this.currentCategory, this.currentQuestion).points;
-		if (isCorrect) {
-			userData.points += points;
-			this.room.add("The answer '" + this.currentAnswer + "' was correct! " + this.currentAnswerer.name + " gains " + points + " points to " + userData.points + "!");
-
-			this.currentUser = this.currentAnswerer;
-			this.currentCategory = -1;
-			this.currentQuestion = -1;
-			this.currentAnswerer = null;
-			this.currentAnswer = "";
-			this.isDailyDouble = false;
-
-			if (this.remainingQuestions === 0) this.autostartFinalRound();
-		} else {
-			userData.points -= points;
-			this.room.add("The answer '" + this.currentAnswer + "' was incorrect! " + this.currentAnswerer.name + " loses " + points + " points to " + userData.points + "!");
+		this.curPlayer = this.players[lowest[Math.floor(lowest.length * Math.random())]];
+		this.prevPlayer = this.curPlayer;
+		this.update();
+		this.nextPlayer();
+	}
 
-			this.currentAnswerer = null;
-			this.currentAnswer = "";
-			this.isDailyDouble = false;
+	nextPlayer() {
+		this.room.addRaw(`${this.curPlayer.name} you're up!`);
+	}
 
-			if (this.remainingAnswers === 0) this.skip(this.host);
+	getgrid() {
+		let buffer = "<div class=\"infobox\"><html><body><table align=\"center\" border=\"2\" style=\"table-layout: fixed; width: 100%\"><tr>";
+		for (let i = 0; i < this.categoryCount; i++) {
+			buffer += `<td style="word-wrap: break-word" bgcolor="${BACKGROUND_COLOR}"; height="${HEIGHT}px"; width="30px" align="center"><font color="white">${this.categories[i]}</font></td>`;
```

Comment:
What, no it wouldn't. It only escapes the part inside `${...}`. It doesn't make sense to escape the actual HTML in it.